swig/perl/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+if (ENABLE_PERL)
```

Comment:
do this in the parent CMakeLists.txt

again, both approaches are valid but pick one and stick to it.

Reply:
@mpreisler OK, I'll do that.